frontend/liferay-theme/src/main/webapp/templates/portal_normal.ftl

```diff
@@ -59,6 +66,12 @@
 <div id="wrapper" class="${hide_portlet_edit_decorators_css} ${login_css}">
 	<header id="banner" role="banner">
 		<div id="heading" class="container">
+            <#if expandoAttribute?has_content>
+                <div id="updateMessage">
+                    <span class="closebtn" onclick="clickCounter()">&times;</span>
+                    <strong>Server Update!</strong> Indicates ${expandoAttribute}.
```

Comment:
I am not sure if this default text `<strong>Server Update!</strong> Indicates` is really required.